sources/packages/backend/apps/api/src/route-controllers/cas-supplier/_tests_/cas-supplier.aest.controller.getCASSupplier.e2e-spec.ts

```diff
@@ -63,6 +66,67 @@ describe("CASSupplierAESTController(e2e)-getCASSuppliers", () => {
             siteStatus: savedCASSupplier1.supplierAddress.status,
             addressLine1: savedCASSupplier1.supplierAddress.addressLine1,
             siteProtected: savedCASSupplier1.supplierAddress.siteProtected,
+            errors: null,
+          },
+        ],
+      });
+  });
+
+  it("Should get CAS supplier with and without error for a student when CAS suppliers info is requested for a student.", async () => {
+    // Arrange
+    const savedCASSupplier1 = await saveFakeCASSupplier(db);
+    const student = savedCASSupplier1.student;
+    const savedCASSupplier2 = await saveFakeCASSupplier(
+      db,
+      { student },
+      {
+        initialValues: {
+          supplierStatus: SupplierStatus.ManualIntervention,
+          errors: [
+            "[0034] SIN is already in use.",
+            "[9999] Duplicate Supplier , Reason: [0065]- Possible duplicate exists, please use online form",
```

Comment:
Is it supposed to emulate a real CAS error?
The string does not look formated.

Reply:
its as per the response we got from the API
<img width="1078" alt="image" src="https://github.com/user-attachments/assets/416db86b-5118-4e62-b86a-23a99ea30424">